test/pseudo-tty/ref_and_unref.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const TTY = process.binding('tty_wrap').TTY;
+
+const handle = new TTY(0);
+handle.readStart();
+
+handle.unref();
+handle.ref();
+
+var timeout = setTimeout(function() {
+  console.log('still running');
+  handle.unref();
+}, 10);
+timeout.unref();
```

Comment:
Maybe this is nitpicking but I'd feel better about testing this with an internal timer:

```js
const timers = require('timers');

// ...

const timer = {
  _onTimeout() {
    console.log('still running');
    handle.unref();
  }
};
timers.enroll(timer, 50);
timers._unrefActive(timer)
```